modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-self-node-remediation-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-self-node-remediation-operator_{context}"]
+= About the Self Node Remediation Operator
+
+The Self Node Remediation Operator runs on the cluster nodes and reboots nodes that are identified as unhealthy. The Operator uses the `MachineHealthCheck` controller to detect the health of a node in the cluster. When a node is identified as unhealthy, the `MachineHealthCheck` resource creates the `SelfNodeRemediation` custom resource (CR), which triggers the Self Node Remediation Operator.
```

Comment:
` controller to detect the health of a node `

maybe will be better to say:
` controller to detect the health of a machine`?

Reply:
Outside the scope of this PR, needs a bug